ci-operator/step-registry/upi/libvirt/cleanup/heterogeneous/upi-libvirt-cleanup-heterogeneous-commands.sh

```
@@ -0,0 +1,63 @@
+
+mock-nss.sh virsh -c "${REMOTE_LIBVIRT_URI}" list
+
+set +e
+
+for DOMAIN in $(${VIRSH} list --all --name | grep "${LEASED_RESOURCE}" || true); do
+  ${VIRSH} destroy "${DOMAIN}"
+  sleep 1s
+  ${VIRSH} undefine "${DOMAIN}"
+done
+
+if [[ -n "$(${VIRSH} pool-list | grep ${POOL_NAME} || true)" ]]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)